prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.ts

```diff
@@ -232,7 +232,7 @@ export class SiteManagementPageComponent implements OnInit {
                 : expiryDates;
             }, []);
           this.organizations = organizations;
-          return this.organization = organizations.find((org) => org.id === organization.id);
+          return this.organization = organizations.find((org) => org.id === this.organizationService.organization.id);
```

Comment:
Can you add a comment re: why this is used instead of previous code?  That is, about preserving selected Organization when navigating to Site Management page?

Reply:
Yes, I was testing this section of code and it was not right. I updated it to get rid of the input org id and always use the organization in org-service. 
Also, I updated the UI for single organization. It was showing the empty dropdown menu.